checker/src/main/java/org/checkerframework/checker/optional/OptionalVisitor.java

```diff
@@ -536,7 +537,13 @@ public Void visitVariable(VariableTree tree, Void p) {
       if (ekind.isField()) {
         checker.reportWarning(tree, "optional.field");
       } else if (ekind == ElementKind.PARAMETER) {
-        checker.reportWarning(tree, "optional.parameter");
+        TreePath paramPath = atypeFactory.getPath(tree);
```

Comment:
In a BaseTypeVisitor subclass, you should call `getCurrentPath()` to get the path to the current tree being visited.